x-pack/plugins/cases/public/components/all_cases/multi_select_filter.tsx

```diff
@@ -20,73 +20,103 @@ import {
   useEuiTheme,
 } from '@elastic/eui';
 import { isEqual } from 'lodash/fp';
-import type { FilterOptions } from '../../../common/ui/types';
 import * as i18n from './translations';
 
-const fromRawOptionsToEuiSelectableOptions = (options: string[], selectedOptions: string[]) => {
+type FilterOption<T extends string> = EuiSelectableOption<{
+  key: string;
+  label: T;
```

Comment:
Is it possible for the label to be something else from a `string`?

Reply:
CaseStatuses and CaseSeverities